core/ops_builtin_v8.rs

```diff
@@ -770,8 +770,15 @@ fn op_dispatch_exception(
     scope.terminate_execution();
     return;
   }
+
+  eprintln!("can borrow inspector {:#?}", state.inspector().try_borrow().is_err());
   match state.inspector().try_borrow() {
     Ok(inspector) if !inspector.has_active_sessions() => {
+      eprintln!("terminating execution");
+      scope.terminate_execution();
+    }
+    Ok(inspector) if inspector.has_active_sessions() => {
+      eprintln!("terminating execution2");
```

Comment:
This is the crux of the problem, I need to drill into this code more; it was added https://github.com/denoland/deno/pull/15548 and now I'm not so sure that it was a good change...